webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.html

```diff
@@ -0,0 +1,74 @@
+<!--View sync message overall status-->
+<div>
+	<div class="row">
+		<div class="col-md-8">
+		</div>
+		<div class="col-md-4">
+			<div>
+				<table style="left: 0;">
+					<tr>
+						<td></td>
+						<td colspan="4"></td>
+						<td colspan="2" i18n="@@common-start">Start Date</td>
```

Comment:
Once you change the label below to **Search By Event Date**, I think **Start** if enough without **Date**, same for **End Date**

Reply:
Done